cinnabar-style.yaml

```diff
@@ -3122,191 +2645,7 @@ layers:
-                filter: { landuse_kind: [park,nature_reserve,conservation,golf_course,garden] }
-                draw:
-                    lines:
-                        color: [[14, global.green1_r],[16, global.minor_road3]]
-                        outline:
-                            color: global.green1_r
-                            width: [[14, 0.1px], [15, 0.5px], [18, 2px]]
-                golf_course:
-                    filter: { landuse_kind: golf_course }
-                    draw:
-                        lines:
-                            outline:
-                                color: global.green1_r
-                nature_reserve:
-                    filter: { landuse_kind: nature_reserve }
-                    draw:
-                        lines:
-                            outline:
-                                color: global.green1_r
-                conservation:
-                    filter: { landuse_kind: conservation }
-                    draw:
-                        lines:
-                            outline:
-                                color: global.green1_r
-            in_cemetery_garden:
-                filter: { landuse_kind: [cemetery,garden] }
-                draw:
-                    lines:
-                        outline:
-                            color: global.green2_r
-                            width: [[15, 0px], [16, 0.5px], [18, 3px]]
-            in_zoo:
-                filter: { landuse_kind: zoo }
-                draw:
-                    lines:
-                        outline:
-                            color: global.green1_r
-                            width: [[15, 0px], [16, 0.5px], [18, 3px]]
-            in_university:
-                filter: { landuse_kind: university }
-                draw:
-                    lines:
-                        color: [[14, global.brown1_r],[16, global.minor_road3]]
-                        outline:
-                            color: global.brown1_r
-                            width: [[14, 0px], [16, 0.5px], [18, 2px]]
-            in_hospital:
-                filter: { landuse_kind: hospital }
-                draw:
-                    lines:
-                        color: [[14, global.red1_r],[16, global.minor_road3]]
-                        outline:
-                            color: global.red1_r
-                            width: [[14, 0px], [16, 0.5px], [18, 2px]]
-            labels-path:
-                filter: { $zoom: { min: 17 } }
-                draw:
-                    text-blend-order:
-                        priority: 61
-                        visible: global.text_visible_path
-                        text_source: global.ux_language_text_source
-                        font:
-                            fill: global.text_fill
-                            size: 12px
-                            stroke: { color: global.text_stroke_road4, width: 4 }
-        track:
```

Comment:
I don't see track labels in the new version, please restore. See Coastal Trail near: 16/37.8333/-122.5379